app/views/DataEntryView.js

```diff
@@ -0,0 +1,40 @@
+
+var Template = require('../util/Template');
+var TelemetryData = require('../models/TelemetryData')
+var fs = require('fs');
+var Logger = require('../util/Logger');
+var app_config=require('./config/application_config.js');
+
+
+module.exports = function (Marionette) {
+
+    return Marionette.ItemView.extend({
+        template: Template('DataEntryView'), 
+        className: 'gainsAdjustView', 
+
+        ui: {
+            name_field: "#fileName",
+            comment_field: "#comment",
+            save_button: "#saveComment"
+     
+        },
+      
+        events: {
+            "click #saveComment": "saveData",
+            
+        },
+        
+        saveData: function (event) { 
+            var d=new Date();
+            var loc = "./"+app_config.get('log_dir') +"/data_entries/" + this.ui.name_field.val()+"_"+d.getDate()+".txt";
```

Comment:
I think it should be ```"data_entries/"``` instead of ```/data_entires/``` (the slash at the begining since the value of log_dir is `logs/`